--- ./feeds.conf.default	2025-09-03 20:56:23.545894206 +0800
+++ ./patch/feeds.conf.default	2025-09-03 20:57:29.920942847 +0800
@@ -1,6 +1,6 @@
 src-git packages https://github.com/coolsnowwolf/packages
-#src-git luci https://github.com/coolsnowwolf/luci.git
-src-git luci https://github.com/coolsnowwolf/luci.git;openwrt-23.05
+src-git luci https://github.com/coolsnowwolf/luci.git
+#src-git luci https://github.com/coolsnowwolf/luci.git;openwrt-23.05
 #src-git luci https://github.com/coolsnowwolf/luci.git;openwrt-24.10
 src-git routing https://github.com/coolsnowwolf/routing
 src-git telephony https://github.com/coolsnowwolf/telephony.git