--- ./package/community/helloworld/luci-app-ssr-plus/luasrc/model/cbi/shadowsocksr/client-config.lua	2025-04-25 13:26:52.053774149 +0800
+++ ./patch/client-config.lua	2025-04-26 11:30:56.198527218 +0800
@@ -991,7 +991,7 @@
 o.rmempty = true
 
 o = s:option(DynamicList, "local_addresses", translate("Local addresses"))
-o.datatype = "cidr"
+o.datatype = "string"
 o:depends({type = "v2ray", v2ray_protocol = "wireguard"})
 o.rmempty = true
 
